network-api/networkapi/wagtailpages/pagemodels/base.py

```diff
@@ -468,6 +468,24 @@ class Homepage(FoundationMetadataPageMixin, Page):
         blank=True
     )
 
+    cause_statement = models.CharField(
+        max_length=250,
+        default="",
+    )
+
+    cause_statement_link_text = models.CharField(
+        max_length=80,
+        default="",
+    )
```

Comment:
These don't get values assigned during migration, so they'll need to be `null=True` or `blank=True`, so that `inv migrate` (and `inv new-db`) don't throw an error.

Reply:
I didn't encounter any errors with `inv new-db` or `inv migrate`, and it seems like the new fields in the migration file all have `default=''` and `null=True`. If we add `blank=True` then the field is no longer a required field. I'm wondering what I missed here?

Reply:
looks to work fine now.